src/Sentry.Unity.Editor/SentryPreprocessBuild.cs

```diff
@@ -10,32 +14,39 @@ public class SentryPreprocessBuild : IPreprocessBuildWithReport
 
         public void OnPreprocessBuild(BuildReport report)
         {
-            // var options = AssetDatabase.LoadAssetAtPath<SentryCliOptions>(SentryWindows.SentryCliOptionsAssetPath);
-
             switch (report.summary.platform)
             {
-                case BuildTarget.StandaloneOSX:
-                case BuildTarget.StandaloneLinux64:
-                case BuildTarget.StandaloneWindows:
-                case BuildTarget.StandaloneWindows64:
-                case BuildTarget.tvOS:
                 case BuildTarget.iOS:
-                case BuildTarget.XboxOne: // pdbs?
-                    // platformGroup = Standalone
-                    // sentry-cli upload-dif
-                    break;
-                case BuildTarget.WebGL:
-                    // sentry-cli sourcemaps
-                    break;
-                case BuildTarget.Android:
-                    // sentry-cli proguard
+                    CheckIOSEditorImportSettingsOnWindows(Path.Combine("Packages", SentryPackageInfo.GetName(), "Editor", "iOS",
+                        "Sentry.Unity.Editor.iOS", ".meta"));
                     break;
                 default:
                     // nothing to do
                     break;
             }
-            // file: summary.outputPath
-            // guid: summary.guid
+        }
+
+        internal void CheckIOSEditorImportSettingsOnWindows(string metaFilePath, IApplication? application = null)
+        {
+            application ??= ApplicationAdapter.Instance;
+            if (application.Platform != RuntimePlatform.WindowsEditor)
```

Comment:
Should this be Windows or Linux? This is anywhere one can be developing with Unity outside of Mac, right?

I guess 0.001% are on Linux but...